editor/javascript/EditorApplication.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Component } from "react";
+import ReactDOM from "react-dom";
+
+class EditorApplication extends Component{
+  render() {
+    return(<main> This will be the main application container.</main>);
+  }
+}
+
+export default EditorApplication;
+
+ReactDOM.render(<EditorApplication />, document.querySelector('#editor-application'));
```

Comment:
Personally I tend to like to move this to an "index" file instead of in the Application file. Not a big deal, but having side-effect-y type code with an export can behave a little funny depending on how they are used.